src/components/atoms/Radio/index.tsx

```diff
@@ -1,13 +1,20 @@
-import {ReactNode} from 'react'
+import { ReactNode } from 'react'
+import { useAppSelector } from 'store/hooks'
+import cn from 'classnames'
 import './style.scss'
 type RadioProps = {
   children: ReactNode
+  id: string
+  onClick: (event: any) => void
 }
-export default function Radio({children}: RadioProps) {
+export default function Radio({ children, id, onClick }: RadioProps) {
+  const { selectedCompany } = useAppSelector(state => state.filter)
   return (
-    <label className="radio-label">
-      <input className="radio-input" type="radio" />
-      <span className="radio-checkmark"></span>
+    <label className="radio-label" onClick={onClick}>
+      <input className="radio-input" type="radio" id={id} />
+      <span
+        className={cn('radio-checkmark', selectedCompany === id && 'active')}
```

Comment:
Атомы не должны взаимодействовать с бизнес-логикой.

```suggestion
        className={cn('radio-checkmark', checked && 'active')}
```